azurepipelines/e2e_test/scenarios/ubuntu-20.04-arm64/Bundle-update.py

```diff
@@ -0,0 +1,164 @@
+# -------------------------------------------------------------------------
```

Comment:
you're adding a bunch of files in /ubuntu-20.04-arm64, yet we have a /ubuntu-18.04-arm64 folder.  Is there really no/little overlap that would avoid a bunch of new files to be added?

Reply:
It has to do with the original structure of the scenarios. When we began this we expected the end-to-end scenarios to require much different code. 

We're in desperate need of a refactoring tracked here: https://microsoft.visualstudio.com/OS/_queries/edit/44133514/?triage=true

so we can remove the redundancy. 

My plan is to roll that work together with the new provisioning work for the test scenarios that needs to be done this quarter. 

It's not something that can be addressed as apart of this PR but rather a change in architecture that MUST be made.